funnel/forms/participant.py

```diff
@@ -26,3 +26,28 @@ class ParticipantForm(forms.Form):
 class ParticipantBadgeForm(forms.Form):
     choices = [('', "Badge printing status"), ('t', "Printed"), ('f', "Not printed")]
     badge_printed = forms.SelectField("", choices=[(val_title[0], val_title[1]) for val_title in choices])
+
+
+# TODO: Move to Baseframe
+class ValidFile(object):
+    def __init__(self, allowed_exts=[], message=None):
+        if not message:
+            message = __(u"Please upload a valid file")
+        self.message = message
+        self.allowed_exts = allowed_exts
+
+    def __call__(self, form, field):
+        ext = os.path.splitext(field.data.filename)[1].strip(".")
+        if ext and ext.lower() not in self.allowed_exts:
+            raise wtforms.validators.StopValidation(self.message)
+
+
+class ParticipantImportForm(forms.Form):
+    error_msg = __(u"Please upload a valid CSV file")
+    participant_list = forms.FileField(__("Participant list"),
+        description=u"Expected headers: fullname*, email*, phone, twitter, company, job_title, city",
+        validators=[forms.validators.DataRequired(error_msg), ValidFile(allowed_exts=['csv'], message=error_msg)])
```

Comment:
File input fields can take an `accept` attribute with a list of mimetypes (only). Can we do that with FileField? (Chrome supports extensions as well, but Safari doesn't.)